docs/source/quickstart.mdx

```diff
@@ -1,296 +1,550 @@
+- **Tool name**: Derived from the operation name
+- **Description**: From GraphQL comments or operation descriptions
+- **Parameters**: Mapped from GraphQL variables
+- **Response**: The GraphQL response data
 
-### Advanced Options
+### Defining Custom Tools
 
-<details>
-<summary>Alternative ways to run the MCP Server</summary>
-
-#### Using STDIO Transport
-
-You can run the MCP Server using STDIO transport instead of Streamable HTTP. This is useful for certain environments or when you need more direct control over the server process.
-
-1. Download the binary of the latest version of Apollo MCP Server
-2. Use MCP Inspector to run the server:
-
-    ```yaml title="Config for stdio transport"
-    endpoint: https://thespacedevs-production.up.railway.app/
-    operations:
-      source: local
-      paths:
-      - <absolute-path-to-MCP-example-dir>/operations
-    schema:
-      source: local
-      path: <absolute-path-to-MCP-example-dir>/api.graphql
-    transport:
-      type: stdio
-    ```
+Create new tools by adding GraphQL operation files to your `operations/` directory:
 
-    ```sh
-    npx @modelcontextprotocol/inspector apollo-mcp-server <path to the preceding config>
-    ```
+```graphql
+# operations/GetProducts.graphql
 
-3. Configure Claude Desktop to use STDIO:
+# Retrieves product information by ID
+query GetProducts($productId: ID!) {
+  product(id: $productId) {
+    id
+    name
+    price
+    description
+  }
+}
+```
 
-    ```json
-    {
-      "mcpServers": {
-        "thespacedevs": {
-          "command": "<absolute-path-to-MCP-server-binary>",
-          "args": [
-            "<path to the preceding config>"
-          ]
-        }
-      }
+Apollo MCP Server automatically detects new `.graphql` files and exposes them as tools. No server restart needed!
+
+**Learn more:** [Defining MCP Tools](https://www.apollographql.com/docs/apollo-mcp-server/define-tools)
+
+### Testing Operations in GraphOS Studio
+
+The `rover dev` command includes a built-in GraphQL explorer:
+
+1. Open http://localhost:4000 in your browser
+2. You'll see GraphOS Studio's Sandbox Explorer
+3. Test your operations directly:
+   - Browse your schema
+   - Run queries and mutations
+   - Experiment with variables
+   - View responses
+
+This is invaluable for:
+- Verifying operations before exposing them as MCP tools
+- Understanding your schema structure
+- Debugging query issues
+- Exploring available data
+
+### Testing with MCP Inspector
+
+Before connecting to Claude, test tools with MCP Inspector:
+
+```bash
+npx @modelcontextprotocol/inspector
+```
+
+Connect to `http://127.0.0.1:5050/mcp` and:
+- List available tools
+- Execute tools with test inputs
+- Inspect tool responses
+- Debug parameter mapping
+
+## Step 5: Deploy Your MCP Server
+
+Once you're ready to deploy, Apollo MCP Server can run in any container environment.
+
+### Using the All-in-One Container
```

Comment:
I believe the official name is the Apollo Runtime Container - we should use that instead.